webapp/src/shared/components/Game/Game.tsx

```diff
@@ -53,18 +57,22 @@ const Play = ({ gameId }: GameProps) => {
 
   return (
     <div className="game-multiplayer">
-      <h1 className="game-multiplayer-score heading-bold">
-        {isPlayerOne ? opponentScore : score}
-      </h1>
+      <div className="game-multiplayer-score">
+        <Score
+          playerOneAvatar={{ url: `${AVATAR_EP_URL}/${playerOne!.avatarId}` }}
```

Comment:
can we add avatarX and avatarY, since we have them? Both for playerOne and playerTwo. Thank you!

Reply:
```diff
diff --git a/webapp/src/shared/components/Game/Game.tsx b/webapp/src/shared/components/Game/Game.tsx
index 847e958..e054235 100644
--- a/webapp/src/shared/components/Game/Game.tsx
+++ b/webapp/src/shared/components/Game/Game.tsx
@@ -59,10 +59,18 @@ const Play = ({ gameId }: GameProps) => {
     <div className="game-multiplayer">
       <div className="game-multiplayer-score">
         <Score
-          playerOneAvatar={{ url: `${AVATAR_EP_URL}/${playerOne!.avatarId}` }}
+          playerOneAvatar={{
+            url: `${AVATAR_EP_URL}/${playerOne!.avatarId}`,
+            XCoordinate: playerOne!.avatarX,
+            YCoordinate: playerOne!.avatarY,
+          }}
           playerOneScore={score}
           playerOneUserName={playerOne!.username}
-          playerTwoAvatar={{ url: `${AVATAR_EP_URL}/${playerTwo!.avatarId}` }}
+          playerTwoAvatar={{
+            url: `${AVATAR_EP_URL}/${playerTwo!.avatarId}`,
+            XCoordinate: playerTwo!.avatarX,
+            YCoordinate: playerTwo!.avatarY,
+          }}
           playerTwoScore={opponentScore}
           playerTwoUserName={playerTwo!.username}
         />
```

Reply:
You're right, thanks. Done